scripts/gitlab/build-linux.sh

```diff
@@ -27,7 +27,6 @@ else
   time cargo build --target $CARGO_TARGET --verbose --color=always --release -p evmbin
   time cargo build --target $CARGO_TARGET --verbose --color=always --release -p ethstore-cli
   time cargo build --target $CARGO_TARGET --verbose --color=always --release -p ethkey-cli
-  time cargo build --target $CARGO_TARGET --verbose --color=always --release -p whisper-cli
```

Comment:
Does that mean we no longer provide whisper binaries?

Reply:
Yes.

cc @s3krit